modules/serverless-rn-1-9-0.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-9-0_{context}"]
+//update the <version> to match the filename
+
+= Release Notes for Red Hat {ServerlessProductName} 1.9.0
+
+* {ServerlessProductName} now uses Knative Serving 0.15.0.
+* {ServerlessProductName} now uses Knative Serving Operator 0.15.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.15.0.
+* {ServerlessProductName} uses Knative Eventing 0.15.0.
+* {ServerlessProductName} now uses Knative Eventing Operator 0.15.0.
```

Comment:
Let's remove this because we're already mentioning the common "Knative Operator" a few lines above.